tests/helpers/utils.py

```diff
@@ -249,4 +250,5 @@ def bin_packing_lp(sizes, capacity):
 
 
 def gastrans_lp():
-    return gastrans_model().relax()
+    gastrans_model().relax()
+    return gastrans_model()
```

Comment:
```suggestion
    gastrans_model().relax()
    return gastrans_model()
 
```
Where is the failing test?

Reply:
Ah, it should be something like

```
model = gastrans_model()
return model.relax()
```

but I don't think I understand your comment

Reply:
This would be the same as before. I thought the change is necessary because `relax()` relaxes in place. So the previous version just returned `None` and some test failed, so if there is none, this can just be deleted.

Reply:
Your thought is correct. No test is using it, but it may in the future, hence the fix.

Reply:
Okay, yes, it should basically be as displayed in https://github.com/scipopt/PySCIPOpt/pull/997#discussion_r2094154880, also for the other relaxations.